docs/partials/api.md

```diff
@@ -144,9 +144,9 @@ The `auro-menuoption` element supports the placement of the `disabled` attribute
 
 </auro-accordion>
 
-#### Menu with pre-selected menu option
+#### `selected`
 
-The `auro-menu` element supports a pre-selected option via the `selected` attribute on the `auro-menuoption` element.
+Use the `selected` attribute to pre-select an option or highlight a specific `auro-menuoption` in the `auro-menu`.
```

Comment:
This confuses me a little that there is an "or" in this. `selected` does one specific thing. There is no highlighting an option with these styles that is not selected.

```suggestion
Use the `selected` attribute to pre-select an `auro-menuoption`.
```